docker_base/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# This is a base container we use. It has pgsql, redis and Python 3.5,
+# plus a user named qabel and locales.
+
+FROM ubuntu:16.04
```

Comment:
@rehwanne suggested alpine linux (2 MB), but almost nothing we need (postgres etc.) is packaged for that. Given that Debian stable has no Python 3.5 and stretch is rolling-release, and that we run on Ubuntu in production made it the logical choice. Going from Debian to Ubuntu saved some ~100 MB.